components/TextFields/src/MDCTextField.m

```diff
@@ -537,7 +545,7 @@ - (CGRect)textRectForBounds:(CGRect)bounds {
     clearButtonWidth += 2 * MDCTextInputClearButtonImageBuiltInPadding;
 
     // Clear buttons are only shown if there is entered text or programatically set text to clear.
-    if (self.text.length > 0) {
+    if (self.hasTextContent > 0) {
```

Comment:
hasTextContent is a BOOL - no need to compare with 0 (appears in multiple places).
Tip: "isEmpty" might be a more conventional name for this.

Reply:
Oh. I missed it when changing it I think.